docs/api-docs/apps/guide/apps-02-types.md

```diff
@@ -3,88 +3,92 @@
+## Single Click
+Single-click apps are the recommended app type. They use [OAuth 2.0 Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/) (oauth2.net) flow. Users initiate installation by clicking **Install**:
 
-## Single Click Apps
+![Install App](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-02-types-01.png "Install App")
 
-Single-click apps are listed in our App Marketplace, making them available for installation on all BigCommerce stores. The “single-click” app flow does not mean users install the app in a single click, but rather, the app does not require the customer to install OAuth credentials themselves or configure any settings for a third party service.
+The app requests the user to grant permissions associated to scopes configured for the app in [DevTools](https://devtools.bigcommerce.com/my/apps):
 
-To put an app on the Marketplace, you must first be a BigCommerce partner and have your app approved through the App Submission Process. After becoming a partner and having the app approved you have the option of charging merchants to use the app.
-Single-click apps use the Client ID and Client Secret obtained from the Dev Tools workspace to request an OAuth token during the [single-click app authentication flow.](/api-docs/getting-started/building-apps-bigcommerce/building-apps#building-apps_oauth-summary)
+![App Permissions](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-02-types-02.png "App Permissions")
 
-## Connector Apps
+Once granted, the app can request a permanent `access_token` for making REST API requests on the user's behalf.
 
-Some apps qualify to be listed in the BigCommerce App Marketplace as “Connector” apps. These apps use manual OAuth token creation instead of the single-click app flow.
-While we always recommend and prefer the single-click app flow’s programmatic OAuth exchange, certain use cases might not be compatible.
-
-Use Cases:
+## Connector
+Connector apps use manual OAuth token creation instead of the single-click app flow. Store owner's generate [Store API Credentials](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#revoking-store-api-credentials) manually and enter them into the app's configuration. While single-click apps are recommended, certain use cases might not be compatible:
 - Customized integrations that vary per store.
-- Integrations that do not provide any content for an iframe.
-- Apps that do not provide Web services for OAuth exchange. (For example: offline sync apps.)
-- Apps that do not provide Web or cloud-hosted components for a dashboard.
+- Integrations that do not provide any content for an iFrame.
+- Apps that do not provide web services for OAuth exchange (ex: native order sync app).
+
+## Visibility

```

Comment:
The information below would be better structured as a table.

[List or table](https://intranet.bigcommerce.com/display/SST/Tables#Tables-Listortable?)

Reply:
Use cases is a list that might grow over time. Its better to have in a list because tables with lots of text do not look very good.

Reply:
Having a running list of bullet points that separate the use cases from the original visibility option also does not look good. Let me know if you need help making this into a table.